node/src/components/deploy_acceptor.rs

```diff
@@ -71,6 +80,7 @@ impl DeployAcceptor {
         effect_builder: EffectBuilder<REv>,
         deploy: Box<Deploy>,
         source: Source<NodeId>,
+        responder: Option<Responder<Result<(), Error>>>,
```

Comment:
include in doc comment when it is appropriate to pass None (from peer) vs Some (from rpc).

Reply:
Added comments